test/runner/lib/manage_ci.py

```diff
@@ -3,10 +3,14 @@
 from __future__ import absolute_import, print_function
 
 import os
-import pipes
 import tempfile
 import time
 
+try:
+    from shlex import quote as cmd_quote
+except ImportError:
+    from pipes import quote as cmd_quote
+
```

Comment:
Instead of repeating the import try/except here, just add `cmd_quote` to the `from lib.util import` below.